deployment/gem.rb

```diff
@@ -235,7 +235,7 @@ def ask(question, default)
   task :migrate do
     status = capture(%{
       cd #{latest_release} &&
-      php app/console --env=prod doctrine:migrations:migrate --dry-run --no-interaction
+      #{php_bin} app/console --env=prod doctrine:migrations:migrate --dry-run --no-interaction
```

Comment:
Isn't this going to break when trying to deploy to production, as php_bin is only set in staging.rb?